README.md

```diff
@@ -85,79 +85,89 @@ The task folder in this repo holds an existing site, including multiple html fil
     )
     ```
   4. We now need to make the relevant folders. Create a ```views``` directory within src. This is what you set in the app.js file as the directory to hold the hbs files.
-  5. In src/views create a ```layouts``` directory.
-  6. In src/views create a ```partials``` directory.
+  5. In `./src/views` create a ```layouts``` directory.
+  6. In `./src/views` create a ```partials``` directory.
 
 
 ### Serving the home page
 
-1. Create a ```home.hbs``` in the views directory
-1. Navigate to public/index.html. Copy all the contents to ```home.hbs```.
-1. Delete public/index.html.
-1. In src/views/layouts create a main.hbs file and insert the following code:
+1. Inside `./src/views/layouts`, create ```main.hbs```.
```

Comment:
I wonder at this point whether it's helpful to contextualise these steps by adding something about the relationship between layouts and templates a little more (beyond your line 103), or whether this would be obstructive? Genuine question. This may end up being moot if #15 is addressed.

Reply:
Maybe this would be a good point for a version of Minesh's diagram, with just views / layouts / partials?

Reply:
Yeah OK, let's see how this runs first.